inc/task.class.php

```diff
@@ -2019,6 +2024,21 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
             }
             break;
 
+         case "forcestart":
+            foreach ($ids as $key) {
+               if ($pfTask->getFromDB($key)) {
+                  if ($pfTask->fields['is_active']) {
+                     $pfTask->forceRunning();
+                     $ma->itemDone($item->getType(), $key, MassiveAction::ACTION_OK);
+                  } else {
+                     $ma->itemDone($item->getType(), $key, MassiveAction::ACTION_KO);
```

Comment:
Not sure it should be considered as an error, maybe using `MassiveAction::NO_ACTION` result would be better.